docs/userguide/configuration.rst

```diff
@@ -2806,6 +2806,17 @@ to the AMQP broker.
 
 If this is set to :const:`0` or :const:`None`, we'll retry forever.
 
+``broker_channel_error_retry``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
we should also add version added 5.3 annotation here

Reply:
thanks, added in [0b985dc](https://github.com/celery/celery/pull/7951/commits/0b985dce930ad20abdd3ccb9ee806764515d7f26)